torch/testing/_internal/distributed/rpc/dist_examples/parameter_server.py

```diff
@@ -0,0 +1,108 @@
+# https://github.com/pytorch/examples/blob/master/distributed/rpc/batch/parameter_server.py
+
+import threading
+from datetime import datetime
+
+import torch
+import torch.distributed.rpc as rpc
+import torch.nn as nn
+from torch import optim
+
+batch_size = 20
+image_w = 64
+image_h = 64
+num_classes = 30
+batch_update_size = 3
+num_batches = 6
+
+
+def timed_log(text):
+    print(f"{datetime.now().strftime('%H:%M:%S')} {text}")
+
+
+class BatchUpdateParameterServer(object):
+
+    def __init__(self, batch_update_size=batch_update_size):
+        self.model = nn.Linear(100, num_classes)
```

Comment:
Since we are replacing module with a simple `Linear` layer, this is no longer called `num_classes`. The first two args in a linear layer are `in_features` and `out_features`. See the doc below. 

https://pytorch.org/docs/stable/generated/torch.nn.Linear.html